src/Behat/Testwork/Tester/Runtime/RuntimeSuiteTester.php

```diff
@@ -54,17 +56,29 @@ public function setUp(Environment $env, SpecificationIterator $iterator, $skip)
     /**
      * {@inheritdoc}
      */
-    public function test(Environment $env, SpecificationIterator $iterator, $skip = false)
+    public function test(Environment $env, SpecificationIterator $iterator, $skip = false, $batch)
     {
+
         $results = array();
         foreach ($iterator as $specification) {
-            $setup = $this->specTester->setUp($env, $specification, $skip);
-            $localSkip = !$setup->isSuccessful() || $skip;
-            $testResult = $this->specTester->test($env, $specification, $localSkip);
-            $teardown = $this->specTester->tearDown($env, $specification, $localSkip, $testResult);
+            if ($batch) {
+                foreach ($specification->getScenarios() as $scenario) {
+                    $cmd = 'bin/behat ' . $specification->getFile() . ":" . $scenario->getLine();
```

Comment:
this does not work because you assume a specific path to Behat, which is likely to be wrong in many cases